client/app/components/authors/HeaderContainer.jsx

```diff
@@ -45,19 +46,21 @@ const HeaderContainer = ({
                         </div>
                     )}
                 </div>
-                <div className="right">
-                    <button
-                        className="button button--menu-icon"
-                        aria-label="Menu"
-                        aria-controls="navigation"
-                        type="button"
-                    >
-                        {isMobileMenuOpen
-                            ? <SVG src={IcClose} onClick={() => setIsMobileMenuOpen(false)} />
-                            : <SVG src={IcMenu} onClick={() => setIsMobileMenuOpen(true)} />}
-                    </button>
-                    {renderMenu(true)}
-                </div>
+                { (!author || shouldShowMenu(author, pages)) && (
```

Comment:
This condition is a bit confusing, perhaps we could rename shouldShowMenu to shouldShowAuthorMenu?

Reply:
I added this condition because otherwise the menu on the homepage wouldn't show. I tried to check `if there's no author, we want to show the menu. If there's an author, we should check this conditions`. How do you think we can change it? Maybe I can move `!author` to a const named `hasAuthor`, what do you think about that? @antsgar

Reply:
The logic makes sense, I just think "shouldShowMenu" could be named "shouldShowAuthorMenu", as we will only check this bit of the condition if there is an author.

Reply:
fixed! @antsgar